frontend/javascripts/admin/task/task_create_form_view.js

```diff
@@ -68,46 +70,88 @@ type State = {
 };
 
 export function handleTaskCreationResponse(responses: Array<TaskCreationResponse>) {
+  if (responses.length === 0) {
+    return;
+  }
   const successfulTasks = [];
   const failedTasks = [];
+  let teamName = null;
 
   responses.forEach((response: TaskCreationResponse, i: number) => {
     if (response.status === 200 && response.success) {
-      successfulTasks.push(
-        `${response.success.id},${response.success.creationInfo ||
-          "null"},(${response.success.editPosition.join(",")}) \n`,
-      );
+      if (!teamName) {
+        teamName = response.success.team;
+      }
+      successfulTasks.push(response.success);
     } else if (response.error) {
       failedTasks.push(`Line ${i}: ${response.error} \n`);
     }
   });
-
-  Modal.info({
-    title: `${successfulTasks.length} tasks were successfully created. ${
-      failedTasks.length
-    } tasks failed.`,
-    content: (
-      <div>
-        {successfulTasks.length > 0 ? (
-          <div>
-            Successful Tasks:
-            <pre>
-              taskId,filename,position
-              <br />
-              {successfulTasks}
-            </pre>
-          </div>
-        ) : null}
-        {failedTasks.length > 0 ? (
+  if (successfulTasks.length > 0) {
+    downloadTasksAsCSV(successfulTasks);
+  }
+  if (failedTasks.length > 0) {
+    Modal.info({
```

Comment:
The failed tasks can be quite a lot, too, which is why I'd download them as CSV, too.